crypto/bls/blst/signature.go

```diff
@@ -46,6 +46,36 @@ func SignatureFromBytes(sig []byte) (common.Signature, error) {
 	return &Signature{s: signature}, nil
 }
 
+// MultipleSignaturesFromBytes creates a group of BLS signatures from a LittleEndian 2d-byte slice.
+func MultipleSignaturesFromBytes(multiSigs [][]byte) ([]common.Signature, error) {
+	if features.Get().SkipBLSVerify {
+		return []common.Signature{}, nil
+	}
+	if len(multiSigs) == 0 {
+		return nil, fmt.Errorf("0 signatures provided to the method")
+	}
+	for _, s := range multiSigs {
+		if len(s) != fieldparams.BLSSignatureLength {
+			return nil, fmt.Errorf("signature must be %d bytes", fieldparams.BLSSignatureLength)
+		}
+	}
+	multiSignatures := new(blstSignature).BatchUncompress(multiSigs)
+	if multiSignatures == nil || len(multiSignatures) == 0 {
+		return nil, errors.New("could not unmarshal bytes into signature")
+	}
+	wrappedSigs := make([]common.Signature, 0, len(multiSignatures))
```

Comment:
if the length is indeed supposed to be equal, we could just allocate it `make([]common.Signature, 
 len(multiSignatures))`